tcms/testcases/tests/test_views.py

```diff
@@ -282,6 +284,8 @@ def setUpTestData(cls):
             "product": cls.case_1.category.product.pk,
             "category": cls.case_1.category.pk,
             "default_tester": "",
+            "testing_duration": "",
+            "setup_duration": "",
```

Comment:
Similar comment here. You can clear one of the fields but then make sure to add valid duration for the other one & add asserts for both fields.